doc/ledmon.conf.pod

```diff
@@ -44,7 +44,8 @@ See also the examples section.
 
 B<EXCLUDELIST> - Ledmon will exclude scanning controllers listed on excludelist.
 When allowlist is also set in config file, the excludelist will be ignored.
-The controllers should be separated by comma (B<,>) character.
+The controllers should be separated by comma (B<,>) character. To exclude all
+controllers once, path to the controllers directory is enough.
 The B<XBD Extended Regular Expressions> are supported.  The deprecated name of
```

Comment:
Please remove line describing regular expression. Please describe how it will work.